src/test/app-tenant.yaml

```diff
@@ -23,6 +60,47 @@ spec:
         gateway: tenant
         host: demo-8081
         port: 8081
+  monitor:
+    - selector:
+        app: httpbin
+      targetPort: 3000
+      portName: service
+      description: Pod Monitor
+      kind: PodMonitor
+    - selector:
+        app: httpbin
+      targetPort: 3000
+      portName: service
+      description: Service Monitor Explicit
+      kind: ServiceMonitor
+    - selector:
+        app: httpbin
+      targetPort: 3000
+      portName: service
+      description: Service Monitor Default
+    - portName: "http"
+      selector:
+        app: "example"
+      targetPort: 8080
+      kind: "PodMonitor"
+      authorization:
+        credentials:
+          key: "example-key"
+          name: "example-secret"
+          optional: false
+        type: "Bearer"
+      description: Pod Monitor with Authorization
+    - portName: "http"
+      selector:
+        app: "example"
+      targetPort: 8080
+      authorization:
+        credentials:
+          key: "example-key"
+          name: "example-secret"
+          optional: false
+        type: "Bearer"
+      description: Service Monitor with Authorization
```

Comment:
I think is fine for now just to run through every case in the operator, we'll want to switch this to different app(s) if/when we actually check the health of the monitor targets.